pkgs/development/python-modules/exiv2/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, pkg-config
+, exiv2
+, python3Packages
+, fetchFromGitHub
+, setuptools
+, toml
+, unittestCheckHook
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "exiv2";
+  version = "0.16.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jim-easterbrook";
+    repo = "python-exiv2";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-DX0pg80fOSkWqrIvcye0btZGglnizzM9ZEuVEpnEJKQ=";
+  };
+
+  build-system = [
+    pkg-config
+    setuptools
+    toml
+  ];
+  dependencies = [
+    exiv2
+  ];
+
+  pythonImportsCheck = [ "exiv2" ];
+  nativeCheckInputs = [ unittestCheckHook ];
+  unittestFlagsArray = [ "-s" "tests" "-v" ];
+
+  meta = with lib; {
+    description = "Low level Python interface to the Exiv2 C++ library";
+    homepage = "https://github.com/jim-easterbrook/python-exiv2";
+    changelog = "https://python-exiv2.readthedocs.io/en/release-${version}/misc/changelog.html";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ zebreus ];
+    platforms = platforms.all;
```

Comment:
```suggestion
```
That's wrong. Python defaults to Unix

Reply:
Thanks, didn't know that. I will just apply the suggestion and remove platforms altogether.